client/app/nonComp/components/NonCompTabs.jsx

```diff
@@ -160,6 +168,7 @@ TaskTableTab.propTypes = {
   description: PropTypes.node,
   predefinedColumns: PropTypes.object,
   tasks: PropTypes.array,
+  featureToggles: PropTypes.bool,
```

Comment:
```suggestion
  featureToggles: PropTypes.shape({
    decisionReviewQueueSsnColumn: PropTypes.bool
  }),
```
This will prevent a PropType warning from being thrown since `featureToggles` is an object. Specifying the shape of the object may be overkill since it only contains a single attribute, but it can add a bit more context for devs in the future regarding which props this component utilizes.